src/clj_kondo/impl/linters/keys.clj

```diff
@@ -8,12 +8,13 @@
 (defn- map-without-nils
   "Like map, but returns nil if any mapped value is nil."
   [f coll]
-  (reduce (fn [acc v]
-            (if-let [new-v (f v)]
-              (conj acc new-v)
-              (reduced nil)))
-          []
-          coll))
+  (-> (reduce (fn [acc v]
```

Comment:
Use `some->` to short-cut on nil, because `(persistent! nil)` will cause trouble.

Reply:
Fixed this, good catch.